metrics_utility/automation_controller_billing/helpers.py

```diff
@@ -9,7 +9,12 @@
 from dateutil import parser
 from dateutil.relativedelta import relativedelta
 
-from metrics_utility.exceptions import UnparsableParameter
+from metrics_utility.exceptions import DateFormatError, UnparsableParameter
+
+
+ALLOWED_EPHEMERAL_PATTERN = r'.^\d+(d|day|days|m|mo|month|months)$'
```

Comment:
```suggestion
ALLOWED_EPHEMERAL_PATTERN = r'^\d+(d|day|days|m|mo|month|months)$'
```

otherwise it never matches anything